src/app/pagamentoplanos/service/pagamentoplanos.service.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Router } from 'express';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PagamentoplanosService {
+  public artPlusURL = `${environment.apiURL}`;
+  public token = localStorage.getItem('token');
+  
+  httpOptions = {
+    headers: new HttpHeaders({
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${this.token}`,
+    }),
+  };
+  
+  constructor(
+    private http: HttpClient,
+    private router: Router
+  ) { }
+  
+  insertPagamento(): Observable<any> {
+    return this.http.post(`${this.artPlusURL}` + `v1`, this.httpOptions);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
    return this.http.post(`${this.artPlusURL}` + "v1", this.httpOptions);
```